src/messaging/SessionParameters.h

```diff
@@ -56,6 +65,8 @@ class SessionParameters
         kInteractionModelRevision = 5,
         kSpecificationVersion     = 6,
         kMaxPathsPerInvoke        = 7,
+        kSupportedTransports      = 8,
+        kMaxTCPMessageSize        = 9,
```

Comment:
If this is the quantity called MAX_TCP_PAYLOAD_SIZE in the spec, why is the naming here different, by the way?

Reply:
I believe I had it as MaxTCPPayload at the place in the TCP layer where it was used but it got renamed to MaxTCPMessageSize based on review suggestions. I will put up a spec PR to rename it there so that they are in sync.